apis/v1alpha3/conversion_test.go

```diff
@@ -135,3 +137,11 @@ func CustomStatusNewFieldFuzzer(in *nextver.VSphereVMStatus, c fuzz.Continue) {
 	in.ModuleUUID = nil
 	in.VMRef = ""
 }
+
+func VSphereMachineSpecNewFieldsFuzzer(in *nextver.VSphereMachineSpec, _ fuzz.Continue) {
+	in.PowerOffMode = nextver.VirtualMachinePowerOpModeHard
+}
+
+func VSphereVMSpecFieldsNewFuzzer(in *nextver.VSphereVMSpec, _ fuzz.Continue) {
+	in.PowerOffMode = nextver.VirtualMachinePowerOpModeHard
+}
```

Comment:
These should be removed, because it's hiding the error in the implementation.

Reply:
Is done, can be resolved